gym/envs/toy_text/hotter_colder.py

```diff
@@ -59,10 +64,11 @@ def step(self, action):
         self.guess_count += 1
         done = self.guess_count >= self.guess_max
 
-        return self.observation, reward[0], done, {"number": self.number, "guesses": self.guess_count}
+        return self.observation, reward, done, {"number": self.number, "guesses": self.guess_count}
```

Comment:
I think the change in this line is failing the tests. Specifically, it returns reward as an array now instead of a scalar, and that does not conform to the Env API